requirements_production.txt

```diff
@@ -5,9 +5,9 @@ djangorestframework>=3.2.0,<3.4.0
 html5lib>=0.9,<1.0
 jsonfield>=0.9.19,<1.1
 natsort>=3.2,<4.1
-reportlab>=3.0,<3.3
+PyPDF2>=1.25.0,<1.26
+reportlab>=3.0,<3.4
```

Comment:
@ostcar Do you tested that all reportlab pdfs works with 3.3? Is there a reason to upgrade to 3.3 in our late beta phase (could bring us new regressions) - and we will drop out reportlab for OpenSlides 2.1 (by replacing with jsPDF).